File: app/src/main/java/io/twere/plainpure/data/model/dribbble/realm/ProjectRealm.java
package io.twere.plainpure.data.model.dribbble.realm;

import io.realm.RealmModel;
import io.realm.annotations.PrimaryKey;
import io.realm.annotations.RealmClass;
import java.util.Date;

@RealmClass public class ProjectRealm implements RealmModel {

  @PrimaryKey private long id;

  private String name;

  private String description;

  private int shots_count;

  private Date createdAt;

  private Date updatedAt;

  private UserRealm user;

  public long getId() {
    return id;
  }

  public void setId(long id) {
    this.id = id;
  }

  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }

  public String getDescription() {
    return description;
  }

  public void setDescription(String description) {
    this.description = description;
  }

  public int getShots_count() {
    return shots_count;
  }

  public void setShots_count(int shots_count) {
    this.shots_count = shots_count;
  }

  public Date getCreatedAt() {
    return createdAt;
  }

  public void setCreatedAt(Date createdAt) {
    this.createdAt = createdAt;
  }

  public Date getUpdatedAt() {
    return updatedAt;
  }

  public void setUpdatedAt(Date updatedAt) {
    this.updatedAt = updatedAt;
  }

  public UserRealm getUser() {
    return user;
  }

  public void setUser(UserRealm user) {
    this.user = user;
  }
}

File: app/src/main/kotlin/io/twere/plainpure/data/local/project/ProjectRealm.kt
package io.twere.plainpure.data.local.project

import io.twere.plainpure.data.local.user.UserRealm
import io.realm.RealmModel
import io.realm.annotations.PrimaryKey
import io.realm.annotations.RealmClass
import java.util.Date

@RealmClass
open class ProjectRealm : RealmModel {

  @PrimaryKey
 open var id: Long = 0
 open var name: String = ""
 open var description: String = ""
 open var shotsCount: Int = 0
 open var createdAt: Date? = null
 open var updatedAt: Date? = null
 open var user: UserRealm? = null

}